android/src/main/java/com/klaviyoreactnativesdk/KlaviyoReactNativeSdkModule.kt

```diff
@@ -65,12 +66,12 @@ class KlaviyoReactNativeSdkModule(
   fun registerForInAppForms(configuration: ReadableMap?) {
     UiThreadUtil.runOnUiThread {
       try {
-        val sessionTimeoutDuration = configuration?.getLong("sessionTimeoutDuration")
-        if (sessionTimeoutDuration != null) {
-          Klaviyo.registerForInAppForms(InAppFormsConfig(sessionTimeoutDuration = sessionTimeoutDuration))
-        } else {
-          Klaviyo.registerForInAppForms()
-        }
+        val timeout = configuration?.getDouble("sessionTimeoutDuration")?.seconds
+        Klaviyo.registerForInAppForms(
+          InAppFormsConfig(
+            sessionTimeoutDuration = timeout ?: InAppFormsConfig.DEFAULT_SESSION_TIMEOUT,
+          ),
+        )
       } catch (e: Exception) {
```

Comment:
Two fixes: 
- `getLong` was failing, bc the value was actually a double (woulda thought those were interchangeable but nay). Doesn't really matter to us what type this is as long as its a number. 
- Recently update android SDK to use `Duration` type, so have to convert from `Double` to `Duration`